atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schema.java

```diff
@@ -301,6 +303,35 @@ public Namespace getNamespace() {
         return namespace;
     }
 
+    public void createIndex(KeyValueService kvs, String indexName) {
```

Comment:
What calls this now? (How did this used to work?)

I think Carrino may have intended to remove this function.